.github/workflows/build_and_test.yml

```diff
@@ -192,14 +192,12 @@ jobs:
         # On windows use install + build so build directory is consistent across runs (i.e. don't build from conan cache)
         # Necessary because setting ccache base_dir doesn't currently work on windows
```

Comment:
This comment is now out of date. Can it go, or does it mean we shouldn't remove the install on windows?